service/acme.go

```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/go-acme/lego/v3/lego"
 )
 
+var CertAutoRenewChan = make(chan string)
```

Comment:
We should make these buffered channels. I'd start with a default of 10 for each but we should plan to make this configurable.